src/helm/benchmark/metrics/aci_bench_metrics.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, List
+
+from helm.benchmark.adaptation.adapter_spec import AdapterSpec
+from helm.benchmark.adaptation.request_state import RequestState
+from helm.benchmark.annotation.aci_bench_annotator import ANNOTATOR_MODELS
+from helm.benchmark.metrics.metric import Metric
+from helm.benchmark.metrics.metric_name import MetricName
+from helm.benchmark.metrics.metric_service import MetricService
+from helm.benchmark.metrics.statistic import Stat
+
+
+class ACIBenchMetric(Metric):
+    """Score metrics for ACIBench."""
+
+    def evaluate_generation(
+        self,
+        adapter_spec: AdapterSpec,
+        request_state: RequestState,
+        metric_service: MetricService,
+        eval_cache_path: str,
+    ) -> List[Stat]:
+        assert request_state.annotations
```

Comment:
Is there some way that the commonality between these metrics could be refactored out? See `AnnotationLikertScaleMetric` [here](https://github.com/stanford-crfm/helm/blob/3251a4fa2109568b15e206b754e446591a7fa626/src/helm/benchmark/metrics/annotation_metrics.py#L74) for an example that you could follow.